.github/CODEOWNERS

```diff
@@ -964,3 +964,7 @@
 
 # Onboarding Documentation and Quickstarts
 /doc/DataPlaneCodeGeneration                                       @chunyu3 @pshao25 @lirenhe @annelo-msft
+
+# PRLabel: %Device Registry
```

Comment:
Please move this to the management library section as the first entry after the catch-all (L933) and follow the existing format.  (L933-937 is a good example)

Reply:
The account issues caught by the linter are confirmed to be fixed.